### PR TITLE
Removed Icons dependency, added RTL support to some icons and text input

### PR DESCRIPTION
**Pull request by filbabic**
### 🎯 Goal

We needed to review our components and make sure we support RTL for our users, similar to how XML does.

### 🛠 Implementation details

Mostly updating icons and drawables, pulling out the Icons dependency from our SDK. Added Content based layout direciton to input field.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| Screenshot_1643213098 | Screenshot_1643213020 |

Channels
| XML | Compose |
| --- | --- |
| Screenshot_1643213009 | Screenshot_1643212997 |

Messages
| XML | Compose |
| --- | --- |
| Screenshot_1643213025 | Screenshot_1643213020 |

### 🧪 Testing

1. Run both XML and Compose apps side-by side.
2. Turn on an RTL based language (Arabic works).
3. Make sure to check that both of the apps look more less the same (structurally). There are some UI differences obviously but the core part of the structure, direction, placement etc should be the same.

#### General
- [x] Assigned a person / code owner group (required)
- [ ] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### CODE REVIEW

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/AttachmentsPicker.kt [hunk 1]
```diff
@@ -353,8 +356,15 @@ private fun AttachmentPickerOptions(
             enabled = hasPickedFiles || hasPickedImages,
             onClick = onSendAttachmentsClick,
             content = {
+                val layoutDirection = LocalLayoutDirection.current
+
                 Icon(
-                    modifier = Modifier.weight(1f),
+                    modifier = Modifier
+                        .weight(1f)
+                        .scale(
```
Comment (bychkovdmitry) [inline stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/AttachmentsPicker.kt:364]: Can we extract this somehow, e.g.:

```
public fun Modifier.mirrorRtl(layoutDirection: LayoutDirection): Modifier {
    return scale(
        scaleX = if (layoutDirection == LayoutDirection.Ltr) 1f else -1f,
        scaleY = 1f
    )
}


modifier = Modifier.mirrorRtl(LocalLayoutDirection.current)
```

**Review by bychkovdmitry — APPROVED** [review]
Great! One minor comment. Also, not sure about text that is a mixture of RTL and LTR substrings. Let's return to this later.

**Comment by bychkovdmitry** [comment]
### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [x] Compose sample runs & works
- [x] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [x] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/AttachmentsPicker.kt (hunk 1 above)
Comment (filbabic) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/attachments/AttachmentsPicker.kt:364]: Yeah this is a good idea - I'll create a util!

**Pull request merged**